client/src/components/CompanyProfile.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {
+  Container,
+  Grid,
+  List,
+  Column,
+  Button,
+  Header,
+  Image,
+  Segment,
+  Divider
+} from "semantic-ui-react";
+
+
+class CompanyProfile extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      opportunityTitle: "",
+      contactName: "",
+      location: "",
+      industry_type: [],
+      expiringDate: ""
+    };
+  }
+
+  handleClickEdit = () => {};
+
+  handleClickDelete = () => {};
+
+  render() {
+    return (
+      
+      <div class="ui segment">
+        <Container>
+          <Container text style={{ marginTop: "5em" }} border={{}}>
```

Comment:
I do not think it's great to do inline style? the best here to add classname and style it using this class name, but not Container can take class name?